spec/index.bs

```diff
@@ -2210,57 +2210,46 @@ enum GPUTextureAspect {
                             - |descriptor|.{{GPUTextureViewDescriptor/baseMipLevel}} +
                                 |descriptor|.{{GPUTextureViewDescriptor/mipLevelCount}} is &le;
                                 |this|.{{GPUTexture/[[mipLevelCount]]}}.
-                            - If |this|.{{GPUTexture/[[dimension]]}} is
-                                <dl class="switch">
-                                    : {{GPUTextureDimension/"1d"}}
-                                    :: Let |arrayCount| be |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=].
-
-                                    : {{GPUTextureDimension/"2d"}}
-                                    :: Let |arrayCount| be |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=].
-
-                                    : {{GPUTextureDimension/"3d"}}
-                                    :: Let |arrayCount| be `1`.
-                                </dl>
                             - |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is &gt; 0.
                             - |descriptor|.{{GPUTextureViewDescriptor/baseArrayLayer}} +
                                 |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is &le;
-                                |arrayCount|.
-                            - |descriptor|.{{GPUTextureViewDescriptor/format}} is |this|.{{GPUTexture/[[format]]}}
-                                or is a [=compatible texture format=] with |this|.{{GPUTexture/[[format]]}}.
-                            - If |descriptor|.{{GPUTextureViewDescriptor/dimension}} is
+                                the [$array layer count$] of |this|.
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} is |this|.{{GPUTexture/[[format]]}}.
+                                <div class="issue">Issue: Allow for creating views with compatible formats as well.</div>
+                            - If |descriptor|.{{GPUTextureViewDescriptor/dimension}} is:
                                 <dl class="switch">
                                     : {{GPUTextureViewDimension/"1d"}}
-                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"1d"}}.
-                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+                                    :: |this|.{{GPUTexture/[[dimension]]}} must be {{GPUTextureDimension/"1d"}}.
```

Comment:
nit: Let's probably update every instance of the "is" style in this algorithm, there are a few more at the top (7 I think). And perhaps "conditions are unsatisfied" -> "requirements are unmet" to match?